src/main/io/osd.c

```diff
@@ -1995,6 +2006,8 @@ void pgResetFn_osdConfig(osdConfig_t *osdConfig)
     osdConfig->units = OSD_UNIT_METRIC;
     osdConfig->main_voltage_decimals = 1;
     osdConfig->attitude_angle_decimals = 0;
+
+    memset(osdConfig->customString, 0, ARRAYLEN(osdConfig->customString));
```

Comment:
I think this could be optimized by replacing with `osdConfig->customString[0] = '\0'`. It could optimize the code space usage.